pkg/web_app/lib/src/hoverable.dart

```diff
@@ -108,7 +108,12 @@ void _setEventForPreCodeCopyToClipboard() {
     container.append(feedback);
 
     button.onClick.listen((_) async {
```

Comment:
This seems like the wrong place to fix this..

Should we perhaps just remove `$ ` from the template instead?

Or we could add another data property like `data-text-to-copy="..."`, but magically deciding that texts starting with `$ ` should get special treatment seems liable to introduce other bugs in the future.

Reply:
> Should we perhaps just remove $ from the template instead?

`dart.dev` has the same pattern for shell commands, I think we should follow that.

Reply:
Updated to use data- attribute, PTAL.